src/symbols/symbols_with_dbghelp.cpp

```diff
@@ -449,4 +451,76 @@ namespace dbghelp {
 }
 }
 
+namespace cpptrace {
+namespace experimental {
+/*
+When a module was loaded at runtime with LoadLibrary after SymInitialize was already called,
+it is necessary to manually load the symbols from that module with SymLoadModuleEx.
+
+See "Symbol Handler Initialization" in Microsoft documentation at
+https://learn.microsoft.com/en-us/windows/win32/debug/symbol-handler-initialization
+*/
+void load_symbols_for_file(const std::string& name) {
+    HMODULE module = GetModuleHandleA(name.c_str());
+    if (module == NULL) {
+        std::fprintf(
```

Comment:
Instead of printing it would be good to throw an `internal_error`, printing on error from a library can be problematic.